### DISCUSSION TITLE
Guide simplifié de la mise en place d'une stack elastic sous k8s - Cas de Zerofiltre.tech

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#177] [Announcements]
# Guide simplifié de la mise en place d'une stack elastic sous k8s - Cas de Zerofiltre.tech

Guide simplifié d'installation d'une plateforme d'observabilité  Elastic (Elastic, filebeat et kibana) sous k8s - chez Zerofiltre.tech 

https://zerofiltre.tech/articles/224

**Comment by 1kevinson** [comment]
Très belle stack, je m'en inspirerai pour mes minis apps
